src/Command/Initializer/Profiles/blank/templates/docker-compose.yml.template

```diff
@@ -0,0 +1,18 @@
+version: '3.7'
+
+volumes:
+  logs:
+    driver: local
+
+services:
+  slim:
+    image: php:7-alpine
```

Comment:
let's bump this up to a php 7.4.6 image

Reply:
The 7-x tag is an alias for the latest version of PHP 7.x.y. So now php:7-alpine will pull version 7.4.6.

Reply:
Okay perfect! Sorry the docker images I use are tagged with full versions hence the confusion